docs/specifications.md

```diff
@@ -7,7 +7,7 @@ order: 100
 
 The following versions are available, and the specific version in use will depend on your configuration.
 
-- PHP Version: 7.4, 8.0, 8.1
+- PHP Version: 7.4, 8.0, 8.1, 8.2 See [Altis compatibility chart](docs://guides/updating-php/#altis-compatibility-chart)
```

Comment:
I think we should add an "experimental" tag to PHP 8.2 for now. Thought?

Reply:
The linked compatibility chart notes that both 8.1 and 8.2 are experimental.